sgf-viewer/prepare_data/requirements.txt

```diff
@@ -1,2 +1,2 @@
 # Install from the same directory as this file
-../../go_attack_utils
+-e ../../go_attack_utils
```

Comment:
make it so that when you update `go_attack_utils`, you don't need to re-run `pip install -r requirements.txt`